Update non working links

iamjoker021 opened the pull request:
Most of links were directed to 404 error

I have updated the links in readme.md so that they could direct to desired page

iamjoker021 committed:
Update non working links

The broken-links are updated to working ones

Zircoz reviewed (changes requested):
For internal links, can you please make them relative paths instead ? That will prevent them from breaking if this repo's address or name changes later.

iamjoker021 committed:
Updating relative path instead of links

As  requested I have updated relative path for Contributing.md and LICENSE instead of links

iamjoker021 commented:
I have changed relative path for Contributing.md and LICENSE instead of links

I'm new to GitHub so I didn't know whether there is relative paths exists for forks, stars etc.

Zircoz commented:
thanks for contributing @iamjoker021 !:D
